pycaret/internal/pycaret_experiment/ts_clustering_experiment.py

```diff
@@ -1,13 +1,11 @@
 import pandas as pd
-
-from pycaret.internal.pycaret_experiment.supervised_experiment import (
-    _SupervisedExperiment,
-)
-from pycaret.internal.pycaret_experiment.unsupervised_experiment import _UnsupervisedExperiment
+from pycaret.internal.pycaret_experiment.ts_unsupervised_experiment import _TSUnsupervisedExperiment
 from pycaret.utils.time_series.forecasting.pipeline import _pipeline_transform
 
 
-class _TSClusteringExperiment(_UnsupervisedExperiment):
+class TSClusteringExperiment(_TSUnsupervisedExperiment):
```

Comment:
This should move into the following folder

pycaret --> time_series --> clustering (new folder) and will be called `oop.py` (similar to other experiments).
We will also have a `functional.py` wrapper around this but that can be added once `oop.py` is implemented.